### PR TITLE
Update Readme informations

### PR DESCRIPTION
**Pull request by skoudoro**
small PR to update readme information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+</a>
+<a href="https://pypi.python.org/pypi/fury"><img src="https://img.shields.io/pypi/v/fury.svg"></a>
+<a href="https://anaconda.org/conda-forge/fury"><img src="https://anaconda.org/conda-forge/fury/badges/version.svg"></a>
+<a href="https://codecov.io/gh/fury-gl/fury"><img src="https://codecov.io/gh/fury-gl/fury/branch/master/graph/badge.svg"></a>
+<a href="https://app.codacy.com/app/fury-gl/fury?utm_source=github.com&utm_medium=referral&utm_content=fury-gl/fury&utm_campaign=Badge_Grade_Dashboard"><img src="https://api.codacy.com/project/badge/Grade/922600af9f94445ead5a12423b813576"></a>
+
+</p>
+
+<p align="center">
+  <a href="#general-informations">General Informations</a> •
```
Comment (Garyfallidis) [inline README.md:25]: General Information

**Comment by Nibba2018** [comment]
Hi @skoudoro, I think the following two simulations look more aggressive and eye catchy.

brick-sim2

File: README.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+| <a href="#"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/shaders_sdf.gif" alt="sdf" width="400px"></a>  | <a href="#"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/collides_simulation.gif" alt="Collides simulation" width="400px"></a> | <a href="#"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/physics_bricks.gif" alt="Physics bricks" width="400px"></a> |
+| Ray Marching and Signed Distance Functions |  *Advanced Nano simulation capablities.* | *Interoperability with the [pyBullet](https://pybullet.org/wordpress/) library.*  |
+| <a href="#"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/ui_loader.gif" alt="UI Loader" width="400px"></a>  | <a href="#"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/ui_tab.gif" alt="UI Tabs" width="400px"></a> | <a href="#"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/ui_sliders.gif" alt="Physics bricks" width="400px"></a> |
+| Custom *Loader UI* |  Custom *Tabs UI.* | Custom *Sliders UI.* and more... |
```
Comment (Garyfallidis) [inline README.md:41]: These do not look so exciting. Keep only the Custom *Tabs UI*. Is the custom loader operational? We should add the dragon too. There is a gif in our chatroom by Lenix

File: README.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+| *Network Visualization*          | *Swarming/flocking simulation based on simple boids rules*  |  *Easy shader effect integration.*  |
+| <a href="#"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/shaders_sdf.gif" alt="sdf" width="400px"></a>  | <a href="#"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/collides_simulation.gif" alt="Collides simulation" width="400px"></a> | <a href="#"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/physics_bricks.gif" alt="Physics bricks" width="400px"></a> |
+| Ray Marching and Signed Distance Functions |  *Advanced Nano simulation capablities.* | *Interoperability with the [pyBullet](https://pybullet.org/wordpress/) library.*  |
```
Comment (Garyfallidis) [inline README.md:39]: Just say Particle collisions. Shading of the box of particle collisions looks strange. Assign Nasim to fix this and regenerate the gif.

**Review by Garyfallidis — CHANGES_REQUESTED** [review]
Overall good start. See comments for next steps. Add this too
image Also add a gif with an example of 3D interaction. Generate one from the picking tutorial.

After this is merged we should add PBR and GLFT examples. We should setup a small sprint for that with @guaje.

**Comment by skoudoro** [comment]
thank you @Nibba2018 and @Garyfallidis for the feedback. I will update the desired gif.

@nasimanousheh, Can you generate a new particle collision gif?  After the generation, you can create a pull request [on this repo](https://github.com/fury-gl/fury-communication-assets)

**Comment by skoudoro** [comment]
This PR is ready to go! Just waiting the new gif from @nasimanousheh

**Comment by Garyfallidis** [comment]
I suggest we merge this as is and @nasimanousheh makes another PR for new gifs.

**Comment by Garyfallidis** [comment]
Done!

**Pull request merged**